docs/server/source/appendices/run-with-docker.md

```diff
@@ -30,45 +38,84 @@ Ready to go!
 Let's analyze that command:
 
 * `docker run` tells Docker to run some image
+* `--interactive` keep STDIN open even if not attached
 * `--rm` remove the container once we are done
-* `-v "$HOME/bigchaindb_docker:/data"` map the host directory
+* `--tty` allocate a pseudo-TTY
+* `--volume "$HOME/bigchaindb_docker:/data"` map the host directory
  `$HOME/bigchaindb_docker` to the container directory `/data`;
  this allows us to have the data persisted on the host machine,
  you can read more in the [official Docker
  documentation](https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume)
-* `-t` allocate a pseudo-TTY
-* `-i` keep STDIN open even if not attached
-* `bigchaindb/bigchaindb` the image to use
-* `-y configure` execute the `configure` sub-command (of the `bigchaindb` command) inside the container, with the `-y` option to automatically use all the default config values
+* `bigchaindb/bigchaindb` the image to use. All the options after the container name are passed on to the entrypoint inside the container.
+* `-y configure` execute the `configure` sub-command (of the `bigchaindb`
+ command) inside the container, with the `-y` option to automatically use all the default config values
+* `--dev-allow-temp-keypair` specifies that this is a dev environment and
```

Comment:
We would want a temp keypair created when we set up the configuration file.
Is it really require a `--dev-allow-temp-keypair` option? It can fail even if no conf file is found.